desk/app/groups.hoon

```diff
@@ -338,11 +338,15 @@
   ^-  (unit (unit cage))
   ?+    pole  [~ ~]
       [%x %gangs ~]  ``gangs+!>(xeno)
-      [%x %init ~]  ``noun+!>([groups-light xeno])
+      [%x %init ~]  ``noun+!>([groups-light-ui xeno])
```

Comment:
Looks like this has produced a `[groups-light xeno]` for some time (at least since before the breaking change to `/groups`). Why are we changing this here/now? Won't that potentially run into the same api breakage?

Reply:
while this was always groups-light, previously that type was `(map flag group)` which changed to `(map flag group-ui)` with the api change. this needs to stay changed so it's called something new to reflect that change. this scry is what we use to initialize groups through the %groups-ui agent, and that uses the `group-ui` type.